39/covrebo/app.py

```diff
@@ -0,0 +1,101 @@
+from flask import Flask, jsonify, abort, make_response, request
+
+NOT_FOUND = 'Not found'
+BAD_REQUEST = 'Bad request'
+
+app = Flask(__name__)
+
+items = [
+    {
+        'id': 1,
+        'name': 'laptop',
+        'value': 1000
+    },
+    {
+        'id': 2,
+        'name': 'chair',
+        'value': 300,
+    },
+    {
+        'id': 3,
+        'name': 'book',
+        'value': 20,
+    },
+]
+
+
+def _get_item(id):
+    return [item for item in items if item['id'] == id]
+
+
+def _record_exists(name):
+    return [item for item in items if item["name"] == name]
```

Comment:
not sure if you got this code from us as input but I would expect this to return a `bool`

Reply:
I'm really new and I took the `app.py` code directly from the [Flask example](https://github.com/pybites/blog_code/blob/master/flaskapi/app.py) that was in the challenge.  Do you mean that my tests should be checking for a bool, rather than checking for the specific item?